config/settings.yml

```diff
@@ -565,10 +565,6 @@ hca:
     pass: <%= ENV['hca__ee__pass'] %>
     user: HCASvcUsr
   endpoint: <%= ENV['hca__endpoint'] %>
-  enrollment_system:
-    ee_summary:
-      api_key: ~
-      host: ~
```

Comment:
I think that moving these settings under a `va1010_forms` prefix makes more since, as they will be shared amongst multiple forms. I'm open to suggestions though.